yt/python/yt/wrapper/default_config.py

```diff
@@ -19,6 +19,8 @@
 # pydoc :: default_config :: begin
 
 DEFAULT_WRITE_CHUNK_SIZE = 128 * common.MB
+DEFAULT_GLOBAL_CONFIG_PATH = "/etc/ytclient.conf"
+DEFAULT_USER_CONFIG_PATH = ".yt/config"
```

Comment:
maybe `~/.yt/config` and `expandpath` later? this relative path looks a bit confusing

Reply:
decided to rename it